src/precice/bindings/matlab/+precice/@Constants/private/preciceGateway.cpp

```diff
@@ -0,0 +1,46 @@
+// Gateway MexFunction object
+#include "mex.hpp"
+#include "mexAdapter.hpp"
+#include <iostream>
+#include "precice/SolverInterface.hpp"
+
+using namespace matlab::data;
+using matlab::mex::ArgumentList;
+using namespace precice::constants;
+
+class MexFunction: public matlab::mex::Function {
+private:
+    ArrayFactory factory;
+    void myMexPrint(const std::string text) {
```

Comment:
`myMexPrint` is private and used once. So let's just put it into the below function.